xarray/tests/test_backends.py

```diff
@@ -1134,6 +1134,22 @@ def test_encoding_kwarg_compression(self):
 
         assert ds.x.encoding == {}
 
+    def test_keep_chunksizes_if_no_original_shape(self):
+        orig_ds = Dataset({'x': [1, 2, 3]})
+        chunksizes = (2, )
+        orig_ds.variables['x'].encoding = {
+            'chunksizes': chunksizes
+        }
+
+        with create_tmp_file() as tmp_file:
+            orig_ds.to_netcdf(path=tmp_file)
+
+            nc = nc4.Dataset(tmp_file, mode='r')
+            with open_dataset(tmp_file) as ds:
+                assert_identical(orig_ds, ds)
```

Comment:
Looks like this is allowed on Windows, but the next line is not. Should I be copying the file to test as a workaround?